doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -1,5 +1,8 @@
 # Upgrade Notes
 
+## 10.5.9
```

Comment:
`10.5.9` is already released.

```suggestion
## 10.5.10
```

Reply:
Oops, we also have to fix the last upgrade note.